contrib/registry/etcd/etcd.go

```diff
@@ -38,6 +39,7 @@ type Registry struct {
 type Option struct {
 	Logger       glog.ILogger
 	KeepaliveTTL time.Duration
+	DialTimeout  time.Duration
```

Comment:
Please add comment for all these attributes. You can copy from https://github.com/etcd-io/etcd/blob/release-3.5/client/v3/config.go